language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -90,6 +90,7 @@ genModule mod
         (defSers, refs) = unzip (map (genDefDataType curModName tpls) serDefs)
         header =
             ["// Generated from " <> T.intercalate "/" (unModuleName curModName) <> ".daml"
+            ,"/* eslint-disable @typescript-eslint/no-unused-vars */"
```

Comment:
Do we really need this? If so, I'd be more happy to use the form which only turns off the warning for the next line whenever we need it, as we do it with the `no-namespace` warning.

Reply:
Oops, that was left over when I was testing stuff. Will remove!